core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -143,7 +143,7 @@ package object config {
       .createWithDefault(false)
 
   private[spark] val EVENT_LOG_ALLOW_EC =
-    ConfigBuilder("spark.eventLog.erasureCoding.enabled")
+    ConfigBuilder("spark.eventLog.allowErasureCoding")
```

Comment:
`allow` looks weird, I'd keep this unchanged or `enableErasureCoding`. Do we expect to see more configs starts with `spark.eventLog.erasureCoding`? It seems possible if we want some fine-grained controls over how to do erasure coding.

Reply:
Ok. I'll keep it unchanged.